client/components/data/query-viewers/README.md

```diff
@@ -12,7 +12,7 @@ import { useSelector } from 'react-redux';
 import QueryViewers from 'calypso/components/data/query-viewers';
 import { getViewers } from 'calypso/state/selectors/get-viewers';
 
-export default function ViewersList( { siteId } ) {
+export default function ViewersList() {
 	const siteId = 1;
```

Comment:
Hm, this looks like a mistake, cc @flootr

Reply:
The `<QueryViewers />` component can be removed from Calypso as it's not used anymore as of https://github.com/Automattic/wp-calypso/pull/50330. Looks like I forgot to do that in my PR.